var/spack/repos/builtin/packages/netcdf-c/package.py

```diff
@@ -161,31 +153,31 @@ def configure_args(self):
         cppflags = []
         ldflags = []
         libs = []
-        config_args = []
+
+        config_args = [
+            "--enable-v2",
+            "--enable-utilities",
+            "--enable-static",
```

Comment:
Wouldn't this hardcode in static libraries? Why can't the choice of static libraries be an option to the package?